whois-commons/src/main/java/net/ripe/db/whois/common/dao/VersionInfo.java

```diff
@@ -5,23 +5,18 @@
 
 import java.util.Objects;
 
-public class VersionInfo implements Identifiable, Comparable<VersionInfo> {
-    private final boolean inLast;
-    private final int objectId;
-    private final int sequenceId;
+
+public class VersionInfo extends Serial implements Identifiable, Comparable<VersionInfo> {
+
     private final VersionDateTime timestamp;
-    private final Operation operation;
 
     public VersionInfo(final boolean inLast, final int objectId, final int sequenceId, final long timestamp, final Operation operation) {
-        this.inLast = inLast;
```

Comment:
If versionInfo extends new class Serial, which means serialId is also available to versionInfo? and this way equals method and toString method of versionInfo now is wrongly implemented? Or am I missing something?

Reply:
good point. added serialId to equals and hash methods (but not compareTo, or sorting would be different)